app-modules/billing/src/Core/Repositories/ConfigPlanRepository.php

```
@@ -56,12 +62,18 @@ private function createPlanFromArray(array $plan, string $name): PlanEntity
         );
     }
 
+    /**
+     * @return Collection<string, PlanEntity>
+     */
     public function getPlansFor(string $name = 'user_'): Collection
     {
         return collect($this->all())
             ->filter(fn ($plan, $key): bool => str_starts_with($key, $name));
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$plan'. (undefined)

(UnusedFormalParameter)